Dungeoneer/src/com/interrupt/dungeoneer/entities/Player.java

```diff
@@ -240,6 +238,9 @@
     // Used to act on breaking changes between save versions
     public int saveVersion = -1;
 
+    /** Currently focused object. */
+    public LookAt lookedAtItem = null;
```

Comment:
Probably can shorten this to just `lookedAt` or `lookedAtEntity`? Because it could be more than just an `Item` object.